app/flows/check_travel_during_coronavirus_flow/outcomes/_not_fully_vaxed.erb

```diff
@@ -13,11 +13,22 @@
   <p class="govuk-body">You must:</p>
 
   <ul class="govuk-list govuk-list--bullet">
-    <li><a href="https://www.gov.uk/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england" class="govuk-link">take a pre-departure COVID-19 test</a> within the 3 days before you travel to England</li>
+    <% if calculator.travelling_to_ukraine? %>
+      <% if calculator.multiple_journey? %>
```

Comment:
Ok, so if you are only travelling to Ukraine, the "pre-departure COVID-19 test" bullet isn't shown at all? Got it. It's a bit of head-scratcher isn't?

Reply:
Yeah, sorry - couldn't find a better way 😢